### PR TITLE
New Blog Post: Making VSCode run faster on Chromebooks

### PR DESCRIPTION
**Pull request by AdaRoseCannon**
Adding new blog post: Making VSCode run faster on Chromebooks

### CODE REVIEW

File: content/posts/2019-10-08--making-vscode-run-faster-on-chromebooks/index.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+The project we are using to do this is called [code-server](https://github.com/cdr/code-server):
+[**cdr/code-server**
+*code-server is VS Code running on a remote server, accessible through the browser. Try it out: docker run -it -p…*github.com](https://github.com/cdr/code-server)
+
+Code server is a server which runs the VSCode backend and exposes the front end via http or https. We will run this back-end in crostini then access it through Chrome, so it works entirely offline but through the local browser.
+
+1. Download the latest linux release from [here](https://github.com/cdr/code-server/releases).
+
+1. Extract the code-server binary.
```
Comment (rzr) [inline content/posts/2019-10-08--making-vscode-run-faster-on-chromebooks/index.md]: 2. Should be 2 no ?

File: content/posts/2019-10-08--making-vscode-run-faster-on-chromebooks/index.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+tags: [Visual Studio Code, Chrome OS, Development, Web Development, Linux]
+---
+
+# Making VSCode run faster on Chromebooks
+
+When Linux came to ChromeOS, I was very happy to start using Visual Studio code again as my main text editor. Unfortunately my Chromebook isn’t very fast and Visual Studio code was running very slowly.
+
+This was because the visuals were being run in a Chrome instance on Linux then forwarded to ChromeOS which is an expensive thing to do on a HiDPI screen like the Pixelbook.
+
+This seems like a waste of resources to me considering we are already running a Chrome front end in ChromeOS itself. We can take advantage of this by running a version of VSCode which exposes the front end via a http server we can access from ChromeOS.
```
Comment (rzr) [inline content/posts/2019-10-08--making-vscode-run-faster-on-chromebooks/index.md]: HTTP

File: content/posts/2019-10-08--making-vscode-run-faster-on-chromebooks/index.md [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+
+1. Use chmod +x code-server to make it executable.
+
+I wrote a short script to automate this process since it needs to be repeated each time it is updated. I download the executable to a folder ~/bin which I have added to my path.
+
+
+
+### Running Visual Studio Code
+
+Since we are running it locally and don’t want to make a https certificate we will run it as http, on port 8080.
```
Comment (rzr) [inline content/posts/2019-10-08--making-vscode-run-faster-on-chromebooks/index.md]: HTTP

File: content/posts/2019-10-08--making-vscode-run-faster-on-chromebooks/index.md [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+
+This was because the visuals were being run in a Chrome instance on Linux then forwarded to ChromeOS which is an expensive thing to do on a HiDPI screen like the Pixelbook.
+
+This seems like a waste of resources to me considering we are already running a Chrome front end in ChromeOS itself. We can take advantage of this by running a version of VSCode which exposes the front end via a http server we can access from ChromeOS.
+
+### Getting started
+
+The project we are using to do this is called [code-server](https://github.com/cdr/code-server):
+[**cdr/code-server**
+*code-server is VS Code running on a remote server, accessible through the browser. Try it out: docker run -it -p…*github.com](https://github.com/cdr/code-server)
```
Comment (rzr) [inline content/posts/2019-10-08--making-vscode-run-faster-on-chromebooks/index.md]:
```sh
docker ...
```

Would be more readable

File: content/posts/2019-10-08--making-vscode-run-faster-on-chromebooks/index.md (hunk 1 above)
Comment (AdaRoseCannon) [reply content/posts/2019-10-08--making-vscode-run-faster-on-chromebooks/index.md]: Mark down lists will increment automatically.

`1.` is essentially and <li> in an <ol>

**Comment by AdaRoseCannon** [comment]
The Gatsby build is failing because it is making a request to Medium which is getting blocked on a security check from cloudflare (in front of medium).

**Pull request merged**